system/filesystem/check-tail.yml

```diff
@@ -0,0 +1,105 @@
+---
+type: CheckConfig
+api_version: core/v2
+metadata:
+  name: check-tail
+  namespace: default
+spec:
+  command: check-tail.rb -f /var/log/messages -P error -l 10
```

Comment:
Not sure what -l stands for, but more labels with defaults like in the other examples would be a good approach!